src/bin/release-please.ts

```diff
@@ -16,7 +16,7 @@
 
 import {coerceOption} from '../util/coerce-option';
 import * as yargs from 'yargs';
-import {GitHub, GH_API_URL, GH_GRAPHQL_URL} from '../github';
+import {GitHub, GH_API_URL, GH_GRAPHQL_URL} from '../scms/github';
```

Comment:
I'm ok with the edges of the code knowing about the concrete implementation of the SCM. The binary script here still assumes GitHub is the SCM as it's the only one. In the future, this would use a factory to build the SCM from config.